test/csv/test_row.rb

```diff
@@ -377,4 +377,16 @@ def test_can_be_compared_when_not_a_row
     r = @row == []
     assert_equal false, r
   end
+
+  def test_dig
+    # by index
+    assert_equal(2, @row.dig(1))
+
+    # by header
+    assert_equal(2, @row.dig("B"))
+
+    # if missing
+    assert_nil(@row.dig("Missing"))
+    assert_nil(@row.dig(100))
```

Comment:
Can you add a test for multiple arguments case?

Reply:
I added multiple arguments case.